src/lib.rs

```diff
@@ -26,6 +26,13 @@ mod test;
 pub mod utils;
 mod web;
 
+#[allow(dead_code)]
```

Comment:
Why is this here?

Reply:
There's a lot of unused `const`s since we're not using the `match` based way of looking up interned strings.